### PR TITLE
Fix error with extends in account templates

### LINKED ISSUE #4526: Deprecated template will error out
We've marked the public profile templates as deprecated through a comment to let people know we will be removing them if they are still using them. However, the `{% comment %}` tag at the top of the file will cause them to error out which we don't want.

https://github.com/openlibhums/janeway/blob/bf7683deff1fa5f1ca27781aeb41d50336d17f39/src/themes/OLH/templates/core/accounts/public_profile.html#L1-L7

THis needs to be updated on:

- [x] OLH
- [x] Clean
- [x] Material

A PR is open on Hourglass: https://github.com/openlibhums/hourglass/pull/417

### PR DESCRIPTION
**Pull request by joemull**
Fixes #4526.

### CODE REVIEW

**Review by ajrbyers — APPROVED** [review]

**Review by StephDriver — COMMENTED** [review]
3 templates ahve had the comment removed rather than moved, please confirm that this was intentional.

File: src/themes/clean/templates/core/accounts/public_profile.html
[the remark this answers is not in the document]
Comment (StephDriver) [reply src/themes/clean/templates/core/accounts/public_profile.html:1]: as per office discussion, this was intended.

File: src/themes/material/templates/core/accounts/public_profile.html
[the remark this answers is not in the document]
Comment (StephDriver) [reply src/themes/material/templates/core/accounts/public_profile.html:1]: ditto above

File: src/themes/OLH/templates/core/accounts/public_profile.html
[the remark this answers is not in the document]
Comment (StephDriver) [reply src/themes/OLH/templates/core/accounts/public_profile.html:1]: ditto below

**Review by StephDriver — APPROVED** [review]

**Pull request merged**